common/src/test/java/tachyon/security/LoginUserTest.java

```diff
@@ -128,6 +131,9 @@ public void getCustomLoginUserProvidedByAppTest() throws Exception {
 
     Assert.assertNotNull(loginUser);
     Assert.assertEquals(loginUser.getName(), "tachyon-user");
+
+    // clean up
+    System.setProperty(Constants.TACHYON_SECURITY_USERNAME, "");
```

Comment:
I was also thinking use `@After` at first, but when finding only 2 out of 7 cases need this clean work, just put it in each case.